src/main/java/nl/smallproject/www/techiteasy/models/Television.java

```diff
@@ -30,6 +28,29 @@ public class Television {
     private Boolean voiceControl;
     private Boolean hdr;
 
+//    @OneToOne(cascade = CascadeType.ALL, fetch = FetchType.LAZY)
+//    @JoinColumn(name = "remotecontroller", referencedColumnName = "id", nullable = true)
+//    private RemoteController remoteController;
+
+//    @OneToOne(cascade = CascadeType.ALL, fetch = FetchType.LAZY)
+//    @JoinColumn(name = "remotecontroller", referencedColumnName = "id", nullable = true)
+    @OneToOne(mappedBy = "television")
```

Comment:
De "mappedby"s hadden beter in de andere kant van de relatie gekunt. In de tekst staat bijvoorbeeld ook "een cimodule kan op meerdere televisies" en om televisie owner te maken van de relaties. Dat heb je nu precies niet gedaan. De relaties zijn verder wel goed gelegd. 
Nog een reden om Television owner te maken, is dat de "assignRemoteControllerToTelvision" functionaliteit in de TelvisionController staat. Het is dus een functie van television en is het dus logisch om television de owner te maken